inference-engine/src/preprocessing/cpu_x86_avx2/ie_preprocess_gapi_kernels_avx2.cpp

```diff
@@ -281,22 +503,11 @@ void calcRowLinear_8UC_Impl(std::array<std::array<uint8_t*, 4>, chanNum> &dst,
 
             // vertical pass
```

Comment:
This comment is also can be removed since there nothing else aside verticalPass function call below

Reply:
I would like to keep this comment, if you don't mind. ;-)